src/markdown-pages/install-with-kurl/system-requirements.md

```diff
@@ -11,7 +11,7 @@ title: "System Requirements"
 * Ubuntu 16.04<sup>*</sup> (Kernel version >= 4.15)
 * Ubuntu 18.04
 * Ubuntu 20.04 (Docker version >= 19.03.10)
-* CentOS 7.4<sup>\*</sup>, 7.5<sup>\*</sup>, 7.6<sup>\*</sup>, 7.7<sup>\*</sup>, 7.8<sup>\*</sup>, 7.9, 8.0<sup>\*</sup>, 8.1, 8.2, 8.3, 8.4 (CentOS 8.x requires Containerd)
+* CentOS 7.4<sup>\*</sup>, 7.5<sup>\*</sup>, 7.6<sup>\*</sup>, 7.7<sup>\*</sup>, 7.8<sup>\*</sup>, 7.9<sup>\*</sup>, 8.0<sup>\*</sup>, 8.1<sup>\*</sup>, 8.2<sup>\*</sup>, 8.3<sup>\*</sup>, 8.4<sup>\*</sup> (CentOS 8.x requires Containerd)
```

Comment:
Isn't CentOS 7.9 still supported? EOL is June 30, 2024.

See https://www.openlogic.com/blog/long-term-outlook-centos-7-support.

Reply:
Yeah, 7.9 is still fine and will be for a while yet, odd as that seems given 8.x's death